hc/api/transports.py

```diff
@@ -216,3 +231,28 @@ def notify(self, check):
         }
 
         return self.post(self.channel.value, payload)
+
+class SMS(HttpTransport):
+    def notify(self, check):
+        # Find these values at https://twilio.com/user/account
+        client = Client(settings.ACCOUNT_SID, settings.AUTH_TOKEN)
+        sms = client.api.account.messages.create(
+            to=self.channel.value,
+            from_="+16414355468",
```

Comment:
@philophilo Wouldn't it be best if this was also set in the env variables? What do you think?

Reply:
True, correcting it